src/components/Application/ProgressBar.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { Accordion, Container, Grid, Header, Label, List, Sticky } from 'semantic-ui-react'
+import { TemplateSectionPayload } from '../../utils/types'
+
+interface SectionPage {
+  sectionIndex: number
+  currentPage: number
+}
+
+interface ProgressBarProps {
+  serialNumber: string
+  templateSections: TemplateSectionPayload[]
+  sectionPage?: SectionPage
+}
+
+const ProgressBar: React.FC<ProgressBarProps> = ({
+  serialNumber,
+  templateSections,
+  sectionPage,
+}) => {
+  const childrenPanels = (section: string, totalPages: number) => {
+    const pages = Array.from(Array(totalPages).keys(), (n) => n + 1)
+    return (
+      <List>
```

Comment:
Not a required change for this PR, but an idea of how this can be done in the future:

```
<List link>
...
<List.Item 
...
active={boolean if should be highlighted}
href={link} (instead of to}
...
>{`Page ${number}`}</List.Item> (not as header but as content)
```
With above, might need to add a click handler

Reply:
Added. Thanks